nntrainer/src/neuralnet.cpp

```diff
@@ -143,15 +143,15 @@ unsigned int parseType(std::string ll, InputType t) {
    *            "he_normal"  : He Normal Initialization
    *            "he_uniform"  : He Uniform Initialization
    */
-  std::array<std::string, 150> weight_ini_string = {"lecun_normal", "lecun_uniform", "xavier_normal", "xavier_uniform",
-                                                    "he_normal",    "he_uniform",    "unknown"};
+  std::array<std::string, 7> weight_ini_string = {"lecun_normal", "lecun_uniform", "xavier_normal", "xavier_uniform",
```

Comment:
I think `layer_string` , `weight_ini_string`... should have same case unless there is any reason.

currently,
 `layer_string`:  PascalCase,
 `weight_ini_string`: snake_case,
 `others`: camel_case || lowercase

Reply:
right. I will change them later. Thanks.